src/Util.ts

```diff
@@ -0,0 +1,5 @@
+import {GRID_CUSTOM_CONFIG} from './config';
+export function hide(key: string): boolean {
+    // @ts-ignore
+    return (typeof GRID_CUSTOM_CONFIG[key] === 'undefined' || GRID_CUSTOM_CONFIG[key]['hide'] === 'undefined') ? false : GRID_CUSTOM_CONFIG[key]['hide'];
```

Comment:
Read about [Optional Chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html), and use here for improved readability, e.g. `GRID_CUSTOM_CONFIG?.[key]?.['hide'] ?? false`

Reply:
Tried that already that did not work here , could not catch this undefined without typeof.

Reply:
It should work. Please schedule a meeting to and we'll do it together.